packages/build/tests/install/snapshots/tests.js.md

```diff
@@ -837,34 +832,22 @@ Generated by [AVA](https://ava.li).
           package: '@netlify/plugin-local-install-core'␊
         - inputs: {}␊
           origin: config␊
-          package: /file/path␊
+          package: plugin/main.js␊
     ␊
     > Context␊
       production␊
     ␊
     > Loading plugins␊
-       - /file/path from netlify.toml␊
+       - plugin/main.js@1.0.0 from netlify.toml␊
     ␊
     ────────────────────────────────────────────────────────────────␊
-      Plugin "/file/path" internal error                      ␊
+      Plugin "plugin/main.js" internal error                      ␊
     ────────────────────────────────────────────────────────────────␊
     ␊
       Error message␊
       Error: Could not import plugin:␊
     ␊
     ␊
-      Plugin details␊
-      Package:        /file/path␊
-      Version:        1.0.0␊
-      Repository:     git+https://github.com/netlify/build.git␊
-      Report issues:  https://github.com/netlify/build/issues␊
-    ␊
-      Error location␊
-      While loading "/file/path" from netlify.toml␊
-    STACK TRACE␊
-    ␊
-      Error properties␊
-      { code: 'MODULE_NOT_FOUND' }␊
```

Comment:
Same as above ☝️ https://github.com/netlify/build/pull/2456/files#r600565406

Reply:
I have a fix for this ready to commit.

Reply:
Fixed in https://github.com/netlify/build/pull/2456/commits/a88ab2fc336fcebd61107ed99c2ecc0a4303d7d9.